cardinal/world_context.go

```diff
@@ -62,6 +81,10 @@ func (ctx *worldContext) Logger() *zerolog.Logger {
 	return ctx.logger
 }
 
+func (ctx *worldContext) GetMessageByType(mType reflect.Type) (types.Message, bool) {
```

Comment:
This is only used by the GetMessage in `world_context.go` that as I mention we shouldn't have, so we should remove this.